### PR TITLE
OTEL/S3M release notes draft

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

OTEL/S3M soft launch release notes

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: blog-service/2023-03-28-otel-collector-apps.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+We’re happy to announce our new and improved data onboarding workflow that gets you up and running with infrastructure monitoring in minutes using our OpenTelemetry collector. Out-of-the-box, you can start monitoring host and process data, web servers (like IIS, Nginx), databases (like MySQL, Redis, Cassandra), and other sources. [Learn more](/docs/get-started/quickstart).
+
+The Sumo Logic Distribution for OpenTelemetry, a single unified agent to send Logs, Metrics, Traces, and Metadata, helps simplify and streamline Observability and debugging to improve overall system reliability and efficiency. [Learn more](/docs/send-data/opentelemetry-collector).
+
+<img src={useBaseUrl('img/send-data/opentelemetry-collector/otel-onboarding.gif')} alt="OpenTelemetry collector onboarding flow" />
```
Comment (kimsauce) [inline blog-service/2023-03-28-otel-collector-apps.md:18]:
```suggestion

Note: The new onboarding workflows are only available for new Trial customers at this time. 

<img src={useBaseUrl('img/send-data/opentelemetry-collector/otel-onboarding.gif')} alt="OpenTelemetry collector onboarding flow" />
```

File: blog-service/2023-03-28-otel-collector-apps.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+We’re happy to announce our new and improved data onboarding workflow that gets you up and running with infrastructure monitoring in minutes using our OpenTelemetry collector. Out-of-the-box, you can start monitoring host and process data, web servers (like IIS, Nginx), databases (like MySQL, Redis, Cassandra), and other sources. [Learn more](/docs/get-started/quickstart).
+
+The Sumo Logic Distribution for OpenTelemetry, a single unified agent to send Logs, Metrics, Traces, and Metadata, helps simplify and streamline Observability and debugging to improve overall system reliability and efficiency. [Learn more](/docs/send-data/opentelemetry-collector).
+
```
Comment (kimsauce) [inline blog-service/2023-03-28-otel-collector-apps.md:17]:
```suggestion
```

File: blog-service/2023-03-28-otel-collector-apps.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  - opentelemetry
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+We’re happy to announce our new and improved data onboarding workflow that gets you up and running with infrastructure monitoring in minutes using our OpenTelemetry collector. Out-of-the-box, you can start monitoring host and process data, web servers (like IIS, Nginx), databases (like MySQL, Redis, Cassandra), and other sources. [Learn more](/docs/get-started/quickstart).
```
Comment (kimsauce) [inline blog-service/2023-03-28-otel-collector-apps.md:14]:
```suggestion
We’re happy to announce our new and improved data onboarding workflow that gets you up and running with infrastructure monitoring in minutes using our OpenTelemetry collector. With this update, you can start monitoring host and process data, web servers (like IIS, Nginx), databases (like MySQL, Redis, Cassandra), and other sources out of the box - no manual configuration required. [Learn more](/docs/get-started/quickstart).
```

File: blog-service/2023-03-28-otel-collector-apps.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  - opentelemetry
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+We’re happy to announce our new and improved OpenTelemetry collector data onboarding workflow that gets you up and running with infrastructure monitoring in minutes. With this update, you can start monitoring host and process data, web servers (like IIS, Nginx), databases (like MySQL, Redis, Cassandra), and other sources out of the box - no manual configuration required. [Learn more](/docs/get-started/quickstart).
```
Comment (stacykor) [inline blog-service/2023-03-28-otel-collector-apps.md:14]:
```suggestion
We’re happy to announce a release that saves you configuration time. Our new and improved OpenTelemetry collector data onboarding workflow that gets you up and running with infrastructure monitoring in minutes. With this update, you can start monitoring host and process data, web servers (like IIS, Nginx), databases (like MySQL, Redis, Cassandra), and other sources out of the box - no manual configuration required. [Learn more](/docs/get-started/quickstart).
```

Comment (stacykor) [reply blog-service/2023-03-28-otel-collector-apps.md:14]: Your text is great but I feel like you're burying the lede. This helps the user get started much faster.

**Review by stacykor — DISMISSED** [review]

File: blog-service/2023-03-28-otel-collector-apps.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+The Sumo Logic Distribution for OpenTelemetry, a single unified agent to send Logs, Metrics, Traces, and Metadata, helps simplify and streamline Observability and debugging to improve overall system reliability and efficiency. [Learn more](/docs/send-data/opentelemetry-collector).
+
+Note: The new onboarding workflows are only available for new Trial customers at this time.
```
[the remark this answers is not in the document]
Comment (kimsauce) [reply blog-service/2023-03-28-otel-collector-apps.md:18]: stet - our "note" admonition formatting doesn't render in RSS Feeds

**Pull request merged**